properties/base.py

```diff
@@ -126,6 +138,28 @@ def __new__(mcs, name, bases, classdict):                                  #pyli
             mcs, name, bases, classdict
         )
 
+        # Update the class defaults to include inherited values
+        _defaults = dict()
+        for parent in reversed(newcls.__mro__):
+            _defaults.update(getattr(parent, '_defaults', dict()))
+
+        # Ensure defaults are valid and add them to the class
+        for key, value in iteritems(_defaults):
+            if key not in newcls._props:
+                raise AttributeError(
+                    "Default input '{}' is not a known property".format(key)
+                )
+            try:
+                if callable(value):
+                    newcls._props[key].validate(None, value())
+                else:
+                    newcls._props[key].validate(None, value)
+            except ValueError:
+                raise AttributeError(
+                    "Invalid default for property '{}'".format(key)
+                )
+        newcls._defaults = _defaults
+
```

Comment:
Makes sense. I wrote [something similar](https://github.com/bsmithyman/galoshes/blob/master/galoshes/meta.py#L40) in the past. Can confirm `reversed` is the right way to do this.